verticaltabs.js

```diff
@@ -843,6 +843,25 @@ VerticalTabs.prototype = {
     }
   },
 
+  delayResizeTabs: function (delay) {
+    if (delay <= 0) {
```

Comment:
We're targeting current versions of Firefox, so default parameters would have been fine, but I've already merged it, so let's do that later.  😉